app/components/Exchange/Exchange.jsx

```diff
@@ -251,11 +250,6 @@ class Exchange extends React.Component {
     }
 
     componentWillMount() {
-        if (Apis.instance().chain_id.substr(0, 8) === "4018d784") {
-            GatewayActions.fetchCoins.defer();
-            GatewayActions.fetchBridgeCoins.defer();
```

Comment:
Bridge coins were fetched here in order to enable the "Purchase X" buttons, are you sure it's no longer required?

Reply:
I will do a check again, just to make sure. I may have overlooked it. Good catch.

Reply:
This is fixed as the `GatewayActions.fetchBridgeCoins.defer();` is now called from the new function `updateGatewayBackers`.

Reply:
Test